client/src/components/TopMenu.css

```
@@ -41,4 +42,36 @@
+
+.top-menu-actions::-webkit-scrollbar-thumb {
+  background-color: var(--color-medium);
+  border-radius: var(--spacing-s);
+}
+
+.top-menu-clear-all {
+  border: none;
+  background: none;
+  padding: 0;
+  color: var(--color-lightest);
+  font: 700 15px/normal 'Quicksand', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Quicksand" (font-family-name-quotes)

(font-family-name-quotes)